packages/css/src/components/document-list/document-list.scss

```diff
@@ -55,16 +54,27 @@ dso-responsive-element[small] > .dso-document-list {
   }
 }
 
-dso-responsive-element:not([small]) > .dso-document-list {
+dso-responsive-element[large] > .dso-document-list {
   .dso-document-list-item {
     &-status {
+      display: flex;
       margin: 0;
+
+      .dso-badge {
+        align-self: flex-start;
+        display: inline-flex;
+      }
+
+      a {
+        align-self: flex-end;
+        display: inline-flex;
+        white-space: nowrap; // stylelint-disable-line declaration-property-value-disallowed-list -- no link wrapping
+      }
     }
   }
 
   .dso-document-list-item-open-document {
-    position: absolute;
-    right: 0;
-    bottom: 0;
+    display: inline-flex;
```

Comment:
Ik twijfel een beetje over de gekozen oplossingsrichting: De tekst kan nu niet boven de anchor tonen, de hele rechterzijde is nu als 'kolom' bezet door de open-document anchor. 
Met een float constructie zou er wel nog ruimte zijn voor tekst volgens mij. Het nadeel van float is dat de anchor dan 'onderaan' plakt, maar dat doet de huidige constructie ook. (maar bij flex kan dat denk ik opgelost worden met justify-self oid).